beacon_node/lighthouse_network/src/discovery/enr.rs

```diff
@@ -188,6 +189,18 @@ pub fn build_enr<E: EthSpec>(
         builder.udp6(udp6_port.get());
     }
 
+    // Add EIP 7636 client information
+    if config.send_eip_7636_info {
+        let name = VERSION
+            .split("/")
+            .collect::<Vec<&str>>()
+            .first()
+            .unwrap_or(&"Lighthouse")
+            .to_string();
```

Comment:
How about adding `client_name()` function (which simply returns &"Lighthouse") to the `lighthouse_version` crate and using it here to simplify the code?